src/components/scenes/WalletListScene.js

```diff
@@ -18,6 +18,7 @@ import Gradient from '../../modules/UI/components/Gradient/Gradient.ui'
 import SafeAreaView from '../../modules/UI/components/SafeAreaView/index.js'
 import { WiredProgressBar } from '../../modules/UI/components/WiredProgressBar/WiredProgressBar.ui.js'
 import { getWalletLoadingPercent } from '../../modules/UI/selectors.js'
+import buyMultipleCryptoStyle from '../../styles/components/BuyCryptoStyle.js'
```

Comment:
This shouldn't be a default import. Use braces to import specific object

Reply:
Ok